config/jest/typescriptTransform.js

```diff
@@ -2,6 +2,6 @@
 
 'use strict';
 
-const tsJestPreprocessor = require('ts-jest/preprocessor');
+const tsJestPreprocessor = require('ts-jest');
```

Comment:
Looks like we can remove this file and add
```json
"transform": {
   "^.+\\.tsx?$": "ts-jest",
}
```
into `scripts/utils/createJestConfig.js` config.

Reply:
Would be nice to be able to remove this workaround, but it should be tested thoroughly.
There were some issues in the past regarding the proper resolution in both ejected and non-ejected environments. This wrapper file was a (kinda hacky, but working) way to get around these issues.